seed/docs/templates/docs/faq.html

```diff
@@ -0,0 +1,56 @@
+{% extends "seed/base.html" %}{% load compress %}
```

Comment:
note that we're using the same angular SEED app as registered at `/app/`. Kind of strange, but our only option for now due to current authentication requirements for other django views